articles/universal-login/i18n.md

```diff
@@ -0,0 +1,34 @@
+---
+description: Internationalization in Universal Login
+topics:
+  - universal-login
+  - internationalization
+toc: true
+---
+# Universal Login Internationalization
+
+## Internationalization in the New Universal Login Experience
+
+The New Universal Login Experience is currently localized to:
+
+- English 
+- Spanish
+
+More languages will be added over time, but it is not currently possible for you to add languages yourself.
+
+### Language Selection
```

Comment:
General feedback;

`ui_locales ` has priority over the `Accept-Language` header and applies only to OAuth2. For `samlp` and `wsfed` we will always look at the `Accept-Language` header.

> For example, if your website can only be rendered in English, maybe you want the Universal Login pages to also be rendered in English, regardless the browser's settings.

Can't we use another language as an example, because `en` is our default and current one.